metecho/api/models.py

```diff
@@ -1227,7 +1260,8 @@ class ScratchOrg(
         default=dict, encoder=DjangoJSONEncoder, blank=True
     )
     currently_refreshing_changes = models.BooleanField(default=False)
-    currently_capturing_changes = models.BooleanField(default=False)
+    currently_retrieving_metadata = models.BooleanField(default=False)
+    currently_retrieving_dataset = models.BooleanField(default=False)
```

Comment:
I kept these separate in case a distinction becomes important in the future. For example, I suppose the FE might want to use different terminology for loading messages.